addons/website/static/src/client_actions/configurator/configurator.xml

```diff
@@ -149,33 +149,35 @@
                         <t t-foreach="state.getFeatures()" t-as="feature" t-key="feature_index">
                             <t t-set='isInstalled' t-value="feature.module_state == 'installed'"/>
                             <div class="o_feature_card_container p-2 w-25" t-if="feature.type != 'empty'">
                                 <div t-attf-class="card h-100 {{isInstalled ? 'card_installed' : (feature.selected ? 'border-success' : '')}}" t-on-click="() => state.toggleFeature(feature.id)">
-                                    <div class="card-body py-2">
+                                    <div class="h3 card-body py-3 px-4">
                                         <t t-if="isInstalled">
                                             <i t-attf-class="o_configurator_feature_status fa fa-info-circle text-muted" title="Already installed"/>
                                         </t>
                                         <t t-else="">
                                             <i t-attf-class="o_configurator_feature_status fa {{feature.selected ? 'fa-check-circle text-success' : 'fa-circle-o text-300'}}" />
                                         </t>
-                                        <h5 t-attf-class="card-title d-flex align-items-center {{isInstalled ? 'text-muted' : ''}}">
-                                            <i t-attf-class="me-2 small fa {{feature.icon}} {{isInstalled ? 'text-muted' : (feature.type == 'page' ? 'text-info' : 'text-warning')}}"/>
-                                            <t t-esc="feature.name"/>
-                                        </h5>
-                                        <p class="card-text small text-muted" t-esc="feature.description"/>
+                                        <div t-attf-class="h2 mb-2 card-title d-flex align-items-center {{isInstalled ? 'text-muted' : ''}}">
+                                            <i t-attf-class="me-2 fa {{feature.icon}} {{isInstalled ? 'text-muted' : (feature.type == 'page' ? 'text-info' : 'text-warning')}}"/>
+                                            <h3 class="m-1">
+                                                <t t-esc="feature.name"/>
+                                            </h3>
+                                        </div>
+                                        <p class="h4 card-text text-muted" t-esc="feature.description"/>
                                     </div>
                                 </div>
                             </div>
                         </t>
                     </div>
                 </div>
-                <div class="text-end border-top pt-2">
-                    <button class="btn btn-primary btn-lg ms-3" t-on-click="buildWebsite">Build my website</button>
+                <div class="pt-2">
+                    <button class="btn btn-primary btn-lg btn-build" t-on-click="buildWebsite">Build my website</button>
```

Comment:
btn-build ?

Reply:
> btn-build ?

Sorry, I think was for a test :smile: I just removed it :+1: